tests/test_packages/test_skills/test_weather.py

```diff
@@ -89,6 +89,23 @@ def test_weather(self, pytestconfig):
         )
         assert result.exit_code == 0
 
+        # Load the agent yaml file and manually insert the things we need
+        yaml_path = os.path.join(
+            "vendor", "fetchai", "skills", "weather_station", "skill.yaml"
+        )
+        file = open(yaml_path, mode="r")
+
+        # read all lines at once
+        whole_file = file.read()
+
+        whole_file = whole_file.replace("is_ledger_tx: True", "is_ledger_tx: False")
+
+        # close the file
+        file.close()
+
+        with open(yaml_path, "w") as f:
+            f.write(whole_file)
+
```

Comment:
Since we changed the weather_station skill to is_ledger_tx to be True by default in the test we need to turned to False when we don't want to interact with the ledger. Otherwise it will fail since we don't have a ledger_api.